pkg/fanal/analyzer/os/release/release.go

```diff
@@ -55,6 +55,8 @@ func (a osReleaseAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInp
 			family = types.OpenSUSELeap
 		case "sles":
 			family = types.SLES
+		case "sle-micro", "sl-micro", "sle-micro-rancher":
```

Comment:
Is there any documentation about this? Or is this your experience?

Reply:
its from internal expierence.
- SUSE Linux Enterprise Micro 5.x is "sle-micro"
- there is SUSE Linux Enterprise Micro 6.0 which used "sl-micro"
- And we have a rebranded delivery called SLE Micro for Rancher with that id. "sle-micro-rancher"

I will add respective samples of os-release,

Reply:
It would be great if you could add this to the comment. 
It might help the developers later.